helix-term/src/ui/mod.rs

```diff
@@ -331,7 +327,7 @@ pub mod completers {
                         path.push("");
                     }
 
-                    let path = path.to_str().unwrap().to_owned();
+                    let path = path.to_string_lossy().into_owned();
```

Comment:
This is still part of the completion and it should return None here instead of an invalid lossy result